<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QListWidget" name="pRecordsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pPlayBtnDetected">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>94</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pAnalyseBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>94</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pLoadBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>94</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QListWidget" name="pDetectedRecordsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pModelPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>./model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pSelectModelBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>94</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pStopBtnDetected">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>670</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pInitAnalyserBtn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Init Analyser</string>
   </property>
  </widget>
  <widget class="QLabel" name="pStatusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>801</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="pNotDetectedRecordsList">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pPlayBtnNotDetected">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>670</y>
     <width>94</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pStopBtnNotDetected">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>670</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pOverlap">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="pSplitPerSeconds">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>15</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Split per [s]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Overlap [s]</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1081</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>1061</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>269</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="pFindClass"/>
       </item>
       <item>
        <widget class="QCheckBox" name="pChckBoxOnlyAccepted">
         <property name="text">
          <string>Only accepted</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="pClasses" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
